.github/workflows/test-runner-resiliency-ichang.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will build and run a workspace manager resiliency test against the terra-wsmtest namespace
```

Comment:
It can be removed. There is now an updated section on [https://github.com/DataBiosphere/terra/blob/main/docs/dev-guides/personal-environments.md](https://github.com/DataBiosphere/terra/blob/main/docs/dev-guides/personal-environments.md) with detailed instructions on how to set things up to run resiliency tests.